tests/mail/tests.py

```diff
@@ -40,48 +48,105 @@
     HAS_AIOSMTPD = False
 
 
-class HeadersCheckMixin:
+def message_from_bytes(s):
+    """email.message_from_bytes() using modern email.policy.default."""
+    return _message_from_bytes(s, policy=policy.default)
+
+
+def message_from_binary_file(fp):
+    """email.message_from_binary_file() using modern email.policy.default."""
+    return _message_from_binary_file(fp, policy=policy.default)
+
+
+class MailTestsMixin:
```

Comment:
Also, can you split it into more commits, e.g.
- Added MailTestsMixin ...
- Added more comments to the ...
- Improved test coverage of ...

etc. This will be really hard to review :no_mouth:

Reply:
Yes, I will split the commits back up to simplify review—thanks for the suggestion.

Are we OK with multiple commits in this single PR for this purpose? (I don't think it would be workable to have several interdependent PRs.)

Reply:
Related question: tests/mail/tests.py is a huge file covering several different aspects of django.core.mail. How would you feel about splitting out a test_message.py for django.core.mail.message, tests_backends.py for django.core.mail.backends.*, etc.? (As a separate commit, or even a separate PR later?)